common/app/conf/switches/ABTestSwitches.scala

```diff
@@ -105,4 +105,14 @@ trait ABTestSwitches {
     sellByDate = new LocalDate(2020, 7, 30),
     exposeClientSide = true
   )
+
+  Switch(
+    ABTests,
+    "ab-commercial-inline1-headings",
+    "Test inline1 ads to not be placed right after h2 headings",
+    owners = Seq(Owner.withGithub("ioanna0")),
+    safeState = On,
```

Comment:
Does `safeState` have to be `On`? Most of our tests have safeState as `Off` since this restores default behaviour.

Reply:
Thanks for pointing it out, have changed it to Off